<launch>

  <node pkg="nodelet" type="nodelet" name="Espros" args="load depth_image_proc/point_cloud_xyz" output="screen">
    <remap from="image_rect" to="espros_distance/image_raw"/>  <!-- Use raw image for efficiency -->
    <remap from="points"     to="espros_distance/points" />
  </node>

</launch>
